geometry_msgs/Point[] map_points
---
geographic_msgs/GeoPoint[] lla_points
